### ISSUE TITLE
Logging not working in Android Studio Emulator

### ISSUE THREAD

**Issue by smhmic** [RodrigoSMarques/flutter_branch_sdk#340]
I am expecting to see network requests being made by the Branch SDK every time the app is opened or an event that is tracked in the app is triggered, whether through a Branch link or triggering the events organically within the app. But we only see minimal logs. The light green are print statements in our code placed just before we call `FlutterBranchSdk.trackContentWithoutBuo()`.
![image](https://github.com/RodrigoSMarques/flutter_branch_sdk/assets/884057/914cda9a-3b68-45d3-a100-8edaf3e0cbb4)

I followed Branch support instructions to:
 - Run the Integration Validator with no errors.
 - Confirm we are using Branch's Flutter SDK 7.3.0. Branch support said:
     > "this points to the Android SDK version being used in the version of Flutter you are using is what is causing these issues with the logs not appearing (logging issue on Android SDK v5.8.0 was [fixed in v5.8.1](https://github.com/BranchMetrics/android-branch-deep-linking-attribution/releases/tag/5.8.1))."

   ![image](https://github.com/RodrigoSMarques/flutter_branch_sdk/assets/884057/ef761ba0-bef5-48ee-a7d0-544ead437191)
 - [Enable logging](https://help.branch.io/developers-hub/docs/android-testing#enable-logging). 
   ![image](https://github.com/RodrigoSMarques/flutter_branch_sdk/assets/884057/9a740165-3175-4b64-9925-485fcf13ab53)
   ![image](https://github.com/RodrigoSMarques/flutter_branch_sdk/assets/884057/a394242d-297a-449a-93cd-f67d6fa0725f)
  I tried with `useTestKey` both on and off, and the output looks the same, except for this log entry:
   ```
   BranchSDK: enableTestMode has been changed. It now uses the test key but will not log or randomize the device IDs. If you wish to enable logging, please invoke enableLogging.
   ```

**Test device:**
 - Flutter: v3.19.5
 - OS: Android 14.0 (U) - API 34
 - Device: Android Studio Emulator 34.1.20-11610631
 - Model: sdk_gphone64_arm64

**Comment by RodrigoSMarques** [comment]
I have identified the issue. Change in log behavior in the latest versions of the Branch SDK.

I will release a fix in the next few days.

**Comment by RodrigoSMarques** [comment]
Fixed Release 8.0.3
